### ISSUE TITLE
Update VS Code plugin with new version of solc compiler

### ISSUE THREAD

**Issue by dima24kmagic** [arthurshoppy/remix-light#5]
Please push changes to the vs code plugin, it seems to be old one still

**Comment by smakni** [comment]
Any update on this please ?

**Comment by arthurshoppy** [comment]
I'll look into using the remote solc compiler so it doesn't need to be updated for each new compiler version.

**Comment by MiletoCarmelo** [comment]
I'm getting this error : 

ParserError: Source file requires different compiler version (current compiler is 0.7.4+commit.3f05b770.Emscripten.clang) - note that nightly builds are considered to be strictly less than the released version

I'm new with solidity, How could I change the compiler version ? is it related to this subject ?

**Comment by cryptoflop** [comment]
@MiletoCarmelo yes, you're trying to compile a solidity file where the solc version is higher than the compiler packaged into this extension.

**Comment by OraclePi** [comment]
@cryptoflop any specific solution? I've tried to replace the solc manually but found nowhere to modify

**Comment by OraclePi** [comment]
@cryptoflop any specific solution? I've tried to replace the solc manually but found nowhere to modify , thanks!

**Comment by 1702KAI** [comment]
Compiler is outdated. Would be great if you can update it to the current version ; )
